src/context.rs

```diff
@@ -123,24 +143,90 @@ impl<A: Actor> Context<A> {
             broadcast_receiver,
```

Comment:
Should this function (attach) be only in the starting phase? It might make sense to add extra actors dynamically if the workload increases, for instance

Reply:
I am unsure about that as I've never used multiple actors on a single `Context`.

Technically, I think we can expose the function for any lifecycle stage.